### PR TITLE
o!ET2: New Article

### PR DESCRIPTION
**Pull request by fajar13k**
- Add osu! European Tournament 2 article
- Add osu! European Tournament 2 to Tournaments index page
- Fix oSEAT2's organisation description

---

### CODE REVIEW

File: wiki/Tournaments/o!ET/2/en.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+5. 32 teams per bracket (96 over the entire tournament) will participate in the tournament. We may NOT increase this limit, but we may shrink it if needed. However, that does not mean only 32 teams per bracket may sign up (see the point below).
+6. Seeding will not be done using PP Average from each team, but using a Qualifiers stage. This Qualifiers stage will also kick out any team that did not make top 32 for their bracket during the Qualifiers Stage.
+7. Only players that are from Europe may play. This includes any player the account of which is located in any of the countries stated on [this list](https://www.countries-ofthe-world.com/countries-of-europe.html "List of countries in Europe").
+8. Winners from the previous editions will not get any disadvantage regarding their own line-up: they are free to have the same as last edition.
+9. Players may not sign-up as a free-agent: if players wish to be scouted by other teams/players, a `#free-agents` text chat will be opened on our Discord server.
+10. Mappool format will go as follows:
+    - Qualifiers: 2 NM/1 HD/1 HR/1 DT.
```
Comment (WalterToro) [inline wiki/Tournaments/o!ET/2/en.md]: I would use full names: Nomod, Hidden, Hardrock, DoubleTime and FreeMod

File: wiki/Tournaments/o!ET/2/en.md [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+tags:
+- osu! European Tournament 2
+- o!ET 2
+- o!ET2
+---
+# osu! European Tournament 2
+
+![o!ET 2 Logo](img/logo.png)
+
+The **osu! European Tournament 2** (***o!ET 2***) is a European players' double-elimination two-on-two osu! tournament hosted by ![][flag_FR] [SiYes](/users/8868144), ![][flag_FR] [Planatek](/users/4869601), ![][flag_FR] [Tigzick](/users/6745742), ![][flag_FR] [Amezurys](/users/5207167), and ![][flag_FR] [TLQ\_Yoshii](/users/7157133). It is the second installment of the osu! European Tournament.
```
Comment (TPGPL) [inline wiki/Tournaments/o!ET/2/en.md]: is an European
delete players'

File: wiki/Tournaments/o!ET/2/en.md [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 1 year of osu!supporter tag, unique profile badge |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 4 months of osu!supporter tag |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 2 months of osu!supporter tag |
+
+## Organisation
+
+The osu! European Tournament 2 Is run by various community members.
```
Comment (TPGPL) [inline wiki/Tournaments/o!ET/2/en.md]: decapitalize is

File: wiki/Tournaments/o!ET/2/en.md [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+  - [dark cat - BUBBLE TEA (feat. juu & cinders) (Hobbes2) \[Fluttering\]](/beatmapsets/769917#osu/1618800)
+  - [jippusu - Mushikui Saikede Rhythm (Amamiya Yuko) \[Chewin\]](/beatmapsets/87547#osu/240730)
+- Hidden
+  - [Negentropy (a.k.a. Team Grimoire) - ouroVoros (Firika) \[Firika's Another\]](/beatmapsets/707164#osu/1557405)
+- HardRock
+  - [Pendulum - The Island (MaxNRG official remix) (Pachiru) \[Insane\]](/beatmapsets/881384#osu/1842852)
+- DoubleTime
+  - [3L - Spring of Dreams (impossiblexu) \[Karen\]](/beatmapsets/27875#osu/97338)
+
+#### Midle Bracket
```
Comment (TPGPL) [inline wiki/Tournaments/o!ET/2/en.md]: Middle

**Pull request merged**